docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The following components are displayed in the interface of the {ProductName} ({ProductShortName}) extension:
+
+* The Run configuration pane is to the right of the Activity bar. The Run configuration pane displays the applications and files analyzed by MTA and any issues uncovered by the analysis. The Run configuration pane includes a folder icon named *Report*, which you can click to open the MTA Report in your browser.
```

Comment:
I don't think the pane has a real name. It seems to be used for different things. You might just want to call it the "left pane" or wherever it is.